kolibri/plugins/coach/assets/src/views/common/tables/ReportsLearnersTable.vue

```diff
@@ -141,9 +142,17 @@
       anyTimeSpent() {
         return this.entries.some(entry => !isUndefined(entry.statusObj.time_spent));
       },
+      // Presumes this is a Lesson report table as exerciseId is only present in lesson routes
+      showTries() {
+        return (
+          this.anyTries &&
+          this.contentMap[this.$route.params.exerciseId]?.options?.modality === Modalities.QUIZ
+        );
+      },
       anyTries() {
         return this.entries.some(entry => !isUndefined(entry.statusObj.tries));
       },
+      // Presumes this is a Lesson report table as quizId is only present in quiz routes
```

Comment:
This got me a bit confused initially, thought this might help future-me (or someone else)

Reply:
So, what you're doing as your approach is using a combination of the modality and the params to find where you are. If there's an exerciseId, you're in a lesson report table, and if it's a quizId, the same exercise is a "quiz". And if you're in a lesson, check the modality to see if it's a practice quiz, in which case show the tries, otherwise, nothing. Right?

Reply:
Yep!